ansible/vagrant/.gitignore

```diff
@@ -1 +1,2 @@
+/.vagrant/
```

Comment:
You don't need the leading slash, but I don't care enough to ask you to change it.